dashboard/README.md

```diff
@@ -1,70 +1,90 @@
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### [server](server/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+The source for an NPM express server that's used in local developer mode to reflect
+the server `/api/v1/endpoints` API call to a remote server. This enables local
+debugging against a real Pbench Server.
 
-### `npm test`
+### [src](src/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+The Pbench Dashboard Javascript source plus additional CSS/LESS and artifacts.
 
-### `npm run build`
+### [src/assets](src/assets/)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### [src/modules](src/modules/)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+`modules` directory has all containers(patent layouts) and components(react components). 
 
-### `npm run eject`
+### [src/utils](src/utils/)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+`utils` directory has all helper/utility scripts.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Setup
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+1. Install NodeJS - [offical setup guide](https://nodejs.org/en/download/package-manager/)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+2. Clone the [Pbench repo](https://github.com/distributed-system-analysis/pbench)
 
-## Learn More
+3. Install all the npm packages.
+Navigate to `dashboard` directory and run following command 
+	```bash
+	$ npm install
+	```
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+	> NOTE:
+Create `.env` file in the root directory(`/dashboard/`) and declare the environment variable `PBENCH_SERVER`.
```

Comment:
I think it's probably important to point out that this is needed *only* for local developer testing. If you're just doing a `build` and deploy, it's meaningless.

Perhaps everything in this section after the `npm install` should be separated into a new "Development and test" section?

Reply:
Updated the Note to be specific for development and test